### PR TITLE
Add Maya 2017 (PySide2) compatibility

### PR DESCRIPTION
**Pull request by BigRoy**
Similar to #68 addresses Maya 2017 compatibility but does so without the need of `Qt.py` and instead uses a simple try-except statement in the form of:

```python
try:
    from PySide2 import QtGui, QtWidgets
except ImportError:
    from PySide import QtGui
    from PySide import QtGui as QtWidgets
```

Unlike #68 this does not change `capture` into a package, it remains a single module.

Tested in Maya 2016 and Maya 2017.

### CODE REVIEW

**Comment by mottosso** [comment]
Awesome!

I'm happy to merge this, let me know when you're done.

File: capture.py [hunk 1]
```diff
@@ -11,6 +11,12 @@
 from maya import cmds
 from maya import mel
 
+try:
+    from PySide2 import QtGui, QtWidgets
+except ImportError:
+    from PySide import QtGui
+    from PySide import QtGui as QtWidgets
```
Comment (mottosso) [inline capture.py]: Let's make this `QtWidgets = QtGui` instead. I keep reading this as though you meant to import QtCore, or something different from the above line.

Comment (BigRoy) [inline capture.py]: Good point. Done with https://github.com/abstractfactory/maya-capture/pull/72/commits/5f903c7536ff62598b035a668037b8b4c88d3e10

**Comment by mottosso** [comment]
Thanks, I'm happy with this.

**Comment by mottosso** [comment]
Version up as well please.

**Comment by BigRoy** [comment]
Also, looking at it now. The update statement at the bottom, shouldn't that apply to Maya 2017 too?

**Comment by mottosso** [comment]
Yes, looks like we need a `if > 2015`

**Comment by BigRoy** [comment]
Shall I do a minor version bump, or patch version bump?

**Comment by mottosso** [comment]
I'd go with a minor, as it's quite significant feature-wise.

**Comment by mottosso** [comment]
> I would go for the first of these two since it requires less maintenance for future versions of Maya.

Looks good to me, haven't seen that before!

**Comment by tokejepsen** [comment]
Thanks @BigRoy, great work :)

**Pull request merged**